frontend/pages/hosts/details/cards/Software/SelfService/SelfService.tsx

```diff
@@ -122,9 +122,12 @@ const SoftwareSelfService = ({
                   </div>
                   <div className={`${baseClass}__items`}>
                     {data.software.map((s) => {
-                      // TODO: update this if/when we support self-service app store apps
-                      const uuid =
-                        s.software_package?.last_install?.install_uuid || "";
+                      let uuid =
+                        s.software_package?.last_install?.install_uuid ??
+                        s.app_store_app?.app_store_id;
```

Comment:
I think this should be the uuid of the install command?

Reply:
ah yes thanks for catching this